content/includes/nginxaas-azure/ssl-tls-prerequisites.md

```diff
@@ -12,12 +12,16 @@ NGINXaaS natively integrates with [Azure Key Vault (AKV)](https://azure.microsof
 
   - If using Access Policies for AKV, ensure that your MI has *GET secrets* or higher permissions.
 
+- Access to AKV through a public or private endpoint. If public access to AKV needs to be restricted, you can do one of the following:
+   - [Configure Network Security Perimeter]({{< ref "/nginxaas-azure/quickstart/security-controls/certificates.md#configure-network-security-perimeter-nsp" >}})
+   - [Integrate with a private endpoint]({{< ref "/nginxaas-azure/quickstart/security-controls/certificates.md#integrate-with-private-endpoint" >}})
+
```

Comment:
this is kinda the same as the "Restrict Public Access to Key Vault" section on `certificates.md`, but here has 2 options but `certificates.md` has 3 options. Which is correct? Can we put a link here to `certificates.md` to de-dupe the content?